SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -62,13 +70,15 @@ void FSpatialTestSettings::Override(const FString& MapName)
 	}
 
 	// Generated config, applied to generated maps
-	FString GeneratedMapOverridesFilename =
-		GeneratedOverrideSettingsBaseFilename + FPackageName::GetShortName(MapName) + (OverrideSettingsFileExtension);
-	if (FPaths::FileExists(GeneratedMapOverridesFilename))
+	FString GeneratedMapConfigurationFilename;
+	if (GenerateMapConfigurationFilename(MapName, GeneratedMapConfigurationFilename))
 	{
 		// Override the settings from the generated map specific config file
-		Load(GeneratedMapOverridesFilename);
+		Load(GeneratedMapConfigurationFilename);
 	}
+
+	// Add special flag for Spatial functional tests to force load the SpatialGDKFunctionalTests module in new processes
+	GetMutableDefault<ULevelEditorPlaySettings>()->AdditionalLaunchParameters += "-SpatialTest";
```

Comment:
```suggestion
	GetMutableDefault<ULevelEditorPlaySettings>()->AdditionalLaunchParameters += " -SpatialTest ";
```
Might run into trouble if someone concats to this again.
TBH not a fan of this pattern, would prefer an array of options that gets joined at point of use.

Reply:
Yeah, I see how this might be causing problems, however I don't think there is any way to solve this without changing Native code?